src/PKSim.CLI.Core/Services/QualificationRunner.cs

```diff
@@ -112,12 +112,12 @@ public async Task RunBatchAsync(QualificationRunOptions runOptions)
          await _jsonSerializer.Serialize(mapping, config.MappingFile);
          _logger.AddDebug($"Project mapping for '{project.Name}' exported to '{config.MappingFile}'", project.Name);
 
-         var projectFile = Path.Combine(projectOutputFolder, $"{project.Name}{CoreConstants.Filter.PROJECT_EXTENSION}");
+         var projectFile = Path.Combine(config.TempFolder, $"{project.Name}{CoreConstants.Filter.PROJECT_EXTENSION}");
```

Comment:
@Yuri05 Still dumping project and snapshot, however into temp folder so that we can access them if we need for some reason